chromium_src/chrome/browser/ui/browser_navigator.cc

```diff
@@ -30,4 +31,14 @@ bool IsHostAllowedInIncognitoBraveImpl(const base::StringPiece& host) {
 
 }  // namespace
 
+#define BRAVE_ADJUST_NAVIGATE_PARAMS_FOR_URL_1 \
+  BraveAdjustNavigateParamsForURL(params);
+
+#define BRAVE_ADJUST_NAVIGATE_PARAMS_FOR_URL_2     \
+  if (brave::IsTorProfile(profile)) {              \
+    profile = brave::GetTorParentProfile(profile); \
+  }
+
 #include "../../../../chrome/browser/ui/browser_navigator.cc"  // NOLINT
```

Comment:
should be 5 `..`

Reply:
Addressed in https://github.com/brave/brave-core/pull/3319/commits/72247b4a257c947c3a677ac2dce292b29b01140a